### PR TITLE
victoriayerm: bio

### PR DESCRIPTION
**Pull request by victoriayerm**

### CODE REVIEW

File: student-bios/VictoriaYermakova.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Things I love:
 * :tada: theatres
 * :heart: learn something new
 
-We can connect on [LinkedIn](https://www.linkedin.com/in/victoria-yermakova/)
+We can connect on [LinkedIn](https://www.linkedin.com/)
```
Comment (snicoll) [inline student-bios/VictoriaYermakova.md:14]: Hey. Getting a link to your profile was actually a good thing. Any reason why you've removed that?

**Review by snicoll — COMMENTED** [review]
👋

File: student-bios/VictoriaYermakova.md (hunk 1 above)
Comment (victoriayerm) [reply student-bios/VictoriaYermakova.md:14]: Changed! Thanks! Indeed my mistake!

Comment (snicoll) [reply student-bios/VictoriaYermakova.md:14]: @victoriayerm did you push? I don't see an update here? You need to `git push` for your local changes to show up here.

Comment (snicoll) [reply student-bios/VictoriaYermakova.md:14]: Any luck pushing your changes here?

Comment (victoriayerm) [reply student-bios/VictoriaYermakova.md:14]: @snicoll I was trying, but do not understand why it is not working, will try again!

Comment (victoriayerm) [reply student-bios/VictoriaYermakova.md]: @snicoll I pushed and pulled request now, can you see it?

**Review by snicoll — APPROVED** [review]
Great, good job @victoriayerm! Please merge, although you've added `.DS_Store` which could have been covered by `.gitignore` but we'll see that in class.

**Comment by snicoll** [comment]
@victoriayerm hellow? :)

**Comment by victoriayerm** [comment]
@snicoll Finally done, thank you for the reminder! I thought I've done everything))

**Pull request merged**